assets/prototype/domain/eventEditor/ticketsList/ticketPriceCalculator/TicketPriceCalculatorModal.js

```diff
@@ -1,80 +1,141 @@
+import indexOf from 'ramda/src/indexOf';
+import pickBy from 'ramda/src/pickBy';
+import { useEffect, useState } from '@wordpress/element';
+
 import TicketPriceCalculatorForm from './TicketPriceCalculatorForm';
+import useTicketPriceCalculatorFormDecorator from './useTicketPriceCalculatorFormDecorator';
+import useTicketPriceCalculatorFormMutators from './useTicketPriceCalculatorFormMutators';
+import { sortByPriceOrderIdAsc } from '../../../shared/entities/prices/sortingPredicates';
+import useTicketPrices from '../../containers/queries/useTicketPrices';
+
 import FormModal from '../../../shared/FormModal';
 // import useFetchTicketRelations from '../../containers/mutations/useFetchTicketRelations';
+/* const prices = [
+	{
+		id: 'NEW_PRICE',
+		dbId: '',
+		amount: null,
+		desc: '',
+		isBasePrice: false,
+		isDeleted: false,
+		isDefault: false,
+		isDiscount: false,
+		isPercent: false,
+		name: '',
+		priceType: 5,
+		order: 999,
+		wpUser: 1,
+	},
+	{
+		id: 'ABC123',
+		dbId: 1234,
+		amount: 9.99,
+		desc: 'Base Price',
+		isBasePrice: true,
+		isDeleted: false,
+		isDefault: false,
+		isDiscount: false,
+		isPercent: false,
+		name: 'Price ABC',
+		priceType: 1,
+		order: 0,
+		wpUser: 1,
+	},
+	{
+		id: 'DEF456',
+		dbId: 1235,
+		amount: 10,
+		desc: 'just to be nice',
+		isBasePrice: false,
+		isDeleted: false,
+		isDefault: false,
+		isDiscount: true,
+		isPercent: true,
+		name: 'ten % off',
+		priceType: 2,
+		order: 20,
+		wpUser: 1,
+	},
+	{
+		id: 'XYZ890',
+		dbId: 1236,
+		amount: 2.5,
+		desc: 'Just Cuz Fee',
+		isBasePrice: false,
+		isDeleted: false,
+		isDefault: false,
+		isDiscount: false,
+		isPercent: false,
+		name: 'gimme gimme',
+		priceType: 5,
+		order: 50,
+		wpUser: 1,
+	},
+	{
+		id: 'TAX123',
+		dbId: 1237,
+		amount: 10,
+		desc: 'theft',
+		isBasePrice: false,
+		isDeleted: false,
+		isDefault: false,
+		isDiscount: false,
+		isPercent: true,
+		name: 'tax',
+		priceType: 7,
+		order: 70,
+		wpUser: 1,
+	},
+]; */
+const priceFormFields = [
+	'amount',
+	'dbId',
+	'desc',
+	'id',
+	'isBasePrice',
+	'isDiscount',
+	'isPercent',
+	'name',
+	'order',
+	'priceType',
+];
+const NO_INDEX = -1;
+const isPriceFormField = (val, key) => indexOf(key, priceFormFields) > NO_INDEX;
```

Comment:
IMO `includes` is cleaner: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Unless we need `indexOf` for some specific reasons.

Reply:
Also, with `includes` we won't need that `NO_INDEX`